### PR TITLE
Try-Catch for OpenCG import statements

### PR DESCRIPTION
**Pull request by wbinventor**
This wraps the ``opencg`` imports in "summary.py" and "opencg_compatible.py" with a try-catch. In addition, it places the ``opencg`` import in "summary.py" specifically within the ``Summary.make_opencg_geometry()`` which is the only routine which uses OpenCG.

### CODE REVIEW

File: src/utils/openmc/opencg_compatible.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import copy
 
 import numpy as np
-import opencg
+
+try:
+    from openmc.opencg_compatible import get_opencg_geometry
```
Comment (paulromano) [inline src/utils/openmc/opencg_compatible.py]: Copy-paste error? I imagine this should be `import opencg`

File: src/utils/openmc/summary.py [hunk 2]
```diff
@@ -533,6 +532,14 @@ def _read_tallies(self):
 
 
     def make_opencg_geometry(self):
+
+        try:
+            from openmc.opencg_compatible import get_opencg_geometry
```
Comment (paulromano) [inline src/utils/openmc/summary.py]: PEP8 does say imports should all be at the top. That being said, OpenCG needs to remain as an optional dependency, so I'm ok with leaving this here for the time being.

File: src/utils/openmc/opencg_compatible.py (hunk 1 above)
Comment (wbinventor) [inline src/utils/openmc/opencg_compatible.py]: yes indeed, quite embarrassing but it imports ``opencg`` correctly now

File: src/utils/openmc/summary.py (hunk 2 above)
Comment (wbinventor) [inline src/utils/openmc/summary.py]: Yeah I knew that this was breaking PEP8, but I think this is a better way to keep OpenCG an optional dependency than some other rather contrived way of tracking whether a user has OpenCG installed.

**Comment by paulromano** [comment]
Thanks for fixing that!

**Comment by nelsonag** [comment]
Thanks for handling that for me Paul!

**Comment by paulromano** [comment]
Oh haha, didn't notice that you had assigned yourself. This was a pretty simple one; thanks for suggesting the fix!

**Pull request merged**